privacyidea/static/components/token/views/token.containerlist.html

```diff
@@ -0,0 +1,118 @@
+<div uib-pagination ng-show="containerdata.count > containersPerPage"
```

Comment:
The container list page is always displayed, even if the user/admin does not have any rights on containers.
While the list is empty for users, helpdesk admins (without the container rights) can see all the containers.